bin/git-effort

```diff
@@ -5,6 +5,7 @@ above=0
 # if the output won't be printed to tty, disable the color
 test -t 1 && to_tty=true
 color=
+columns=45
```

Comment:
taklen that out. I have another branch where you can set the columns as option on the command line when calling git effort, this was the default setting. however it's not necessary here ...